lms/djangoapps/verify_student/views.py

```diff
@@ -1047,3 +1054,84 @@ def reverification_window_expired(_request):
+        if course is None:
+            raise Http404
+        checkpoint = VerificationCheckpoint.get_verification_checkpoint(course_key, checkpoint_name)
+        if checkpoint is None:
+            raise Http404
+        init_verification = SoftwareSecurePhotoVerification.get_initial_verification(user)
+        if not init_verification:
+            return redirect(reverse('verify_student_verify_later', kwargs={'course_id': unicode(course_key)}))
+        context = {
+            'course_key': unicode(course_key),
+            "course_name": course.display_name_with_default,
+            'checkpoint_name': checkpoint_name,
+        }
+        return render_to_response("verify_student/incourse_reverify.html", context)
+
+    @method_decorator(login_required)
+    def post(self, request, course_id, checkpoint_name):
+        """
+        submits the reverification to SoftwareSecure
+        """
+
+        user = request.user
+        course_key = CourseKey.from_string(course_id)
+        course = modulestore().get_course(course_key)
+        checkpoint = VerificationCheckpoint.get_verification_checkpoint(course_key, checkpoint_name)
+        if checkpoint is None:
+            log.error("Checkpoint is not defined. Could not submit verification attempt for user %s",
+                      request.user.id)
+            context = {
+                'course_key': unicode(course_key),
+                "course_name": course.display_name_with_default,
+                'checkpoint_name': checkpoint_name,
+                "error": True,
+                "errorMsg": "No checkpoint found"
+            }
+            return render_to_response("verify_student/incourse_reverify.html", context)
+        init_verification = SoftwareSecurePhotoVerification.get_initial_verification(user)
+        if not init_verification:
+            log.error("Could not submit verification attempt for user %s", request.user.id)
+            return redirect(reverse('verify_student_verify_later', kwargs={'course_id': unicode(course_key)}))
+
+        try:
+            b64_face_image = request.POST['face_image'].split(",")[1]
+            attempt = SoftwareSecurePhotoVerification(user=request.user)
+            attempt.upload_face_image(b64_face_image.decode('base64'))
+            attempt.photo_id_key = init_verification.photo_id_key
+            attempt.mark_ready()
+
+            attempt.save()
+            attempt.submit()
+
+            checkpoint.add_verification_attempt(attempt)
+            VerificationStatus.add_verification_status(checkpoint, user, "submitted")
+            return HttpResponse(json.dumps("Completed!"), content_type="application/json")
+        except IndexError:
+            log.error("Invalid image data during photo verification.")
+            return HttpResponseBadRequest(_("Invalid image data during photo verification."))
+        except:
+            log.error("Could not submit verification attempt for user %s", request.user.id)
```

Comment:
This should be a `log.exception` so that the raised exception gets logged as well.